sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -181,6 +181,14 @@ static void zePrint(const char *Format, ...) {
   }
 }
 
+// Controls whether device-scope events are used.
+static const bool ZeAllHostVisibleEvents = [] {
+  const auto DeviceEventsStr =
+      std::getenv("SYCL_PI_LEVEL_ZERO_DEVICE_SCOPE_EVENTS");
```

Comment:
Can we name this as SYCL_PI_LEVEL_ZERO_ALL_HOST_VISIBLE_EVENTS?

Reply:
We can, but I like current name more because it conveys Level-Zero event "scope" concept